src/SFA.DAS.Assessor.Functions.UnitTests/Ofqual/OfqualDownloaderTests.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using SFA.DAS.Assessor.Functions.Functions.Ofqual;
+using Moq;
+using SFA.DAS.Assessor.Functions.Domain.OfqualImport.Interfaces;
+using System.Net.Http;
+
+namespace SFA.DAS.Assessor.Functions.UnitTests.Ofqual
+{
+    public class OfqualDownloaderTests
+    {
+        [Test]
+        public void OrganisationsConstructor_CreatesOrganisationsHttpClient()
+        {
+            var mockBlobTransferClient = new Mock<IOfqualDownloadsBlobFileTransferClient>();
+            var mockHttpClient = new Mock<IHttpClientFactory>();
```

Comment:
The factory is being mocked not the the client - variable naming

Reply:
Renamed